pkgs/development/python-modules/magika/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, click
+, fetchPypi
+, numpy
+, onnxruntime
+, poetry-core
+, python-dotenv
+, pythonOlder
+, tabulate
+, tqdm
+}:
+
+buildPythonPackage rec {
+  pname = "magika";
+  version = "0.5.0";
+  pyproject = true;
+  disabled = pythonOlder "3.8";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-r6C7iDCG/o3JEvweQGb4upr+LuHvmNtkwtduZGehCsc=";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = [
+    click
+    numpy
+    onnxruntime
+    python-dotenv
+    tabulate
+    tqdm
+  ];
+
+  pythonImportsCheck = [ "magika" ];
+
+  meta = with lib; {
```

Comment:
`mainProgram` must be set 
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

although I am not sure if we should set it for both, or just the `magika` attribute, i guess which ever is easiest

Reply:
I set it, thanks for the suggestion